src/main/java/nitezh/ministock/domain/StockQuoteRepository.java

```diff
@@ -50,6 +51,7 @@ of this software and associated documentation files (the "Software"), to deal
 
 public class StockQuoteRepository {
 
+    public GlobalWidgetData myData = new GlobalWidgetData();
     private static final ImmutableBiMap<String, String> GOOGLE_SYMBOLS = new ImmutableBiMap.Builder<String, String>()
```

Comment:
`public GlobalWidgetData myData = new GlobalWidgetData();` never used. if you do remove it remove its respective import as well.